samza-core/src/test/java/org/apache/samza/operators/TestMessageStreamImpl.java

```diff
@@ -310,6 +315,57 @@ public void testJoin() {
     assertEquals(rightInputOpSpec, ((JoinOperatorSpec) leftRegisteredOpSpec).getRightInputOpSpec());
   }
 
+  @Test
+  public void testSendToTable() {
+    StreamGraphImpl mockGraph = mock(StreamGraphImpl.class);
+    OperatorSpec inputOpSpec = mock(OperatorSpec.class);
+    MessageStreamImpl<TestMessageEnvelope> source = new MessageStreamImpl<>(mockGraph, inputOpSpec);
+
+    TableSpec tableSpec = new TableSpec();
+    TableImpl table = new TableImpl(tableSpec);
+
+    source.sendTo(table);
```

Comment:
Curious, can we support the following cases?
1) source1.sendTo(table); source2.sendTo(table);
2) source1.join(table, joinFn); source2.join(table, joinFn);

It seems that we allow this DAG in the programming API. We should verify that we support this case, or reject those cases if donot want to support them.

Reply:
Added test case TestLocalTable.testDualStreamTableJoin()